bom/pom.xml

```
@@ -66,7 +66,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>commons-io</groupId>
         <artifactId>commons-io</artifactId>
-        <version>2.6</version>
+        <version>2.7</version>
```

```suggestion
        <version>2.8.0</version>
```

Why upgrade to an outdated release?

---

Because 2.8.0 was released yesterday, after my PR :angry: :man_facepalming:

---

I see:
* Removed **_read_** from InfiniteCircularInputStreamtop
* Some serialVersionUID changes on serializable interfaces.

Not a big problem from my PoV

---

LMAO I just noticed that after commenting as the announce email hadn't even been sent yet until like an hour after I said this!

---

I just learnt a new acronym. Seriously, it's a parallel language :laughing: